DNN Platform/DotNetNuke.Web/Api/Internal/DnnDependencyResolver.cs

```diff
@@ -36,7 +37,8 @@ public DnnDependencyResolver(IServiceProvider serviceProvider)
         /// </returns>
         public IDependencyScope BeginScope()
         {
-            return new DnnDependencyResolver(_serviceProvider.CreateScope().ServiceProvider);
+            var scope = System.Web.HttpContext.Current.GetScope();
```

Comment:
This actually looks like it is the correct way to handle the DI Scope in System.Web

Reply:
@ahoefling I assume this is the note of the dependence on System.Web.  I'm ok with this as we are using System.Web now, so it makes sense, and this is internal to our stuff